image-registry/remove-registry-baremetal.md

```diff
@@ -68,21 +68,31 @@ And later enable the image registry with the desired storage
 - When the image registry operator bootstraps itself, it will inspect the cluster infrastructure platform status.
 - If the platform status type is `baremetal`, the registry will mark itself `Removed`.
 - A separate installation process will be resposible for provisioning storage. For RHV and k8s-native infra, this is expected to be a PVC with the `RWX` access mode.
+- Telemeter should be aware if a cluster's registry has been removed:
+  - Reason codes in the ClusterOperator conditions (`Available` in particular).
+  - Prometheus alerts warning registry has been removed.
 
 ### Risks and Mitigations
 
 #### Image registry is never installed
 
-Lots of components will likely report errors if a customer runs the installer as `baremetal` and does nothing to set up the registry.
+Several components will likely report errors if a customer runs the installer as `baremetal` and does nothing to set up the registry.
 Potential candidates that will break:
-1. Builds which push to an imagestream tag - will error at runtime.
-2. Samples operator - reports degraded if imagestreams fail to import.
-3. Imagestream import with `--reference-policy=local` may always fail.
+
+1. Builds which push to an imagestream tag will error at runtime.
+2. Samples operator imagestreams which import from `registry.redhat.io` will require a pull secret to be usable.
+3. Imagestreams imported with `--reference-policy=local` will not reference the internal registry as expected. `BuildConfig` and `DeploymentConfig` objects which do their own resolution of `ImageStreamTag` will break.
```

Comment:
i don't know that they'll "break", but they won't get a local reference back from the resolution.